DependencyInjection/Configuration.php

```diff
@@ -15,13 +15,35 @@ public function getConfigTreeBuilder()
         $rootNode
             ->children()
                 ->arrayNode('exporter')
+                    ->addDefaultsIfNotSet()
                     ->children()
-                        ->scalarNode('key')->isRequired()->example('generated')->end()
-                        ->scalarNode('mailto')->example('cron@example.com')->end()
-                        ->scalarNode('path')->example('/usr/local/bin:/usr/bin:/bin') ->end()
-                        ->scalarNode('executor')->example('php') ->end()
-                        ->scalarNode('console')->example('app/console') ->end()
-                        ->scalarNode('shell')->example('/bin/sh') ->end()
+                        ->scalarNode('key')
+                            ->cannotBeEmpty()
+                            ->defaultValue('%kernel.name%_%kernel.environment%')
```

Comment:
I don't think this should be a default value. The problem is if you have multiple apps on the same machine they will likely have the same default value which would cause all sorts of problems.